### ISSUE TITLE
Usage of Slack vs more async tools

### ISSUE THREAD

**Issue by antoviaque** [openedx/wg-coordination#111]
Contributor meetup topic imported from https://openedx.atlassian.net/wiki/spaces/COMM/pages/3934289923/2023-11-28+CC+Working+Group+Meeting+Notes

* A lot of activity only happens on Slack, which is hard to follow for anyone not spending their life there - it also encourages synchronicity, and the community’s archive is held up in a proprietary tool
* Should we make a more concerted effort to move to more async tools (and open source), like Discourse?

**Comment by antoviaque** [comment]
Mentioned in https://discuss.openedx.org/t/contributors-meetup-async-update-december-23rd-2023-january-5th-2024/12031/2#notebook_with_decorative_cover-summary-of-responses-3

_"@ali-hugo Has asked if it’s possible for all working group meetings to either be recorded, or generate automatic meeting notes, so that CC’s who aren’t able to attend the meetings can still see what’s been discussed."_

**Comment by jalondonot** [comment]
09/01 Contributor Meetup Update:
- @antoviaque and @jalondonot will elaborate more on the proposal for having a community handbook that could include, among other topics, some guidelines or principles for cooperating and working asynchronously
- @jalondonot will reach out to @feoh to align, if possible, this proposal with any ongoing work with a similar scope

**Comment by antoviaque** [comment]
Also mentioned as an action item: providing documentation and training to help community members to configure their notifications and workflows, to see all the relevant messages - but only the relevant messages. This could be documentation to include in the handbook?

**Comment by antoviaque** [comment]
I have tested transcribing and summarizing a few meetings using an Otter bot:
* [Redwood release planning](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3995107330/Redwood+Meeting+Planning+1+Jan+11+2024): https://otter.ai/u/AN8Sucrf8Achr5PsIal5xySOhfU
* [Core product working group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/4002218000/January+16+2024+-+Meeting+Notes): https://otter.ai/u/t0a5lMqivwHMq_ZcEUENK79fYBw
* Transifex: https://otter.ai/u/j4EiVtSiHU1gTxIJG73PV1efFio
* Marketing working group: https://otter.ai/u/zxG7rHqxpRPg7vCz6bvPlxw85NQ
* Campus working group: https://otter.ai/u/IkmOT8IcZ7XWmO3x_DZBVeg1HyU

**Comment by antoviaque** [comment]
For Otter, one blocking element was the lack of notice of the recording prior to the meeting - it took some back and forth with the Otter support to add it, but now Otter should be sending an email to all participants in the calendar invite before the meeting start, informing of it.

**Comment by antoviaque** [comment]
Discussions are happening on a document, about establishing a position of the Open edX project about AI recording of meetings. @e0d do you think we could post the URL publicly here?

**Comment by antoviaque** [comment]
With the ban of AI tools in meetings, we are a bit back to square one on this.

That said, when we discussed the ban, the counterpart was to ensure that all working group & project meetings would, going forward provide publicly a video recording and transcript. This, by itself, will help a lot folks who want to follow meetings asynchronously. 

@e0d I don't think this has been fully implemented yet, no? Is there anything we can do to put it in place? If that helps we have some automation scripts we use at OpenCraft for automatically downloading Zoom video recordings and uploading them to Google Drive. We could also look into automation for publishing the transcripts.

**Comment by antoviaque** [comment]
We disucssed it during the meeting last week, and established that it would be good to put in place some automation. We might have some existing scripts we put in place for our own usage at OpenCraft (I'll ask around), and we will try to find a core contributor who have some hours to help putting this in place.

**Comment by pkulkark** [comment]
@e0d I'd be happy to help put what @Kelketek shared in place for Axim.

**Comment by e0d** [comment]
Seems useful, though some OpenCraft specific features and some things we would probably want to extend, for example adding some summarization capability.  I'll chat with the team and we can talk about next steps.  We would need to, for example, switch all meetings to use Zoom, which isn't currently the case.

**Comment by antoviaque** [comment]
@e0d Sounds good! And if we can help with bridging that gap, we can look into that too.

**Comment by antoviaque** [comment]
Update: see post from Ed proposing changes that would enable automating everything: https://discuss.openedx.org/t/input-needed-proposed-change-to-working-group-meeting-hosting/13845

**Comment by antoviaque** [comment]
I believe this has been implemented. @ali-hugo I remember that you had trouble finding some of the meeting recordings at some point, right? I think I saw it being corrected since then - still any issue?

**Comment by ali-hugo** [comment]
@antoviaque I had some trouble accessing the latest Core Product recording, but it seems that some meeting ownership settings have been updated since then. I’ll check on this again after the meeting next week and share an update here once I know more.

**Comment by ali-hugo** [comment]
> I believe this has been implemented. @ali-hugo I remember that you had trouble finding some of the meeting recordings at some point, right? I think I saw it being corrected since then - still any issue?

@antoviaque A quick message to confirm that I **was** able to access the recording from yesterday's Core Product meeting, so the issue seems to have been resolved. 👏
